proto/axelar-cork/v1/cork.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+package axelar_cork.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/peggyjv/sommelier/x/axelarcork/types";
+
+message Cork {
+  // call body containing the ABI encoded bytes to send to the contract
+  bytes encoded_contract_call = 1;
+  // the chain ID of the evm target chain
+  uint64 chain_id = 2;
+  // address of the contract to send the call
+  string target_contract_address = 3;
+}
+
+message ScheduledCork {
+  Cork cork = 1;
+  uint64 block_height = 2;
+  string validator = 3;
+  bytes id = 4;
+}
+
+message ScheduledCorks {
+  repeated ScheduledCork scheduled_corks = 1;
+}
+
+message CorkResult {
+  Cork cork = 1;
+  uint64 block_height = 2;
+  bool approved = 3;
+  string approval_percentage = 4;
+}
+
+message CorkResults {
+  repeated CorkResult cork_results = 1;
+}
+
+message CellarIDSet {
+  repeated string ids = 1;
+}
+
+message ChainConfiguration {
+  string name           = 1;
+  uint64 id             = 2;
+  string vote_threshold = 3 [
```

Comment:
Unclear. It's a carryover from Axelar v1, where we had a vote threshold for Ethereum. It could be moved to a Param, but @EricBolten and I couldn't really think of a reason to go in either direction, so left it here. More configuration seems better?